proto/pdpb.proto

```diff
@@ -295,3 +295,35 @@ message Error {
     optional BootstrappedError     bootstrapped = 2;
     optional StoreIsTombstoneError is_tombstone = 3;
 }
+
+service PlacementService {
+    rpc Tso(Request) returns (Response) {}
```

Comment:
> I think it's better to be rpc Tso(TsoRequest) returns (TsoResponse).

I agree.

> If TsoRequest, we should embed RequestHeader in TsoRequest

It seems to be the best solution, but it is redundant for the old rpc.